openedx_webhooks/github/dispatcher/actions/utils.py

```diff
@@ -85,7 +91,7 @@ def _update_commit_status_for_cla(url, payload):
     return data
 
 
-def cla_status_on_pr(pull_request: PrDict) -> Optional[str]:
+def cla_status_on_pr(pull_request: PrDict) -> Optional[Dict[str, str]]:
```

Comment:
Hmm, this type change is related to the PR: I wanted to pass around a full four-key dict as the status instead of just "success" or "failure", so that I could customize the detailed messages.

There are other typing changes in here that aren't related, I did them so that I could use mypy to help with the refactoring.